astropy/visualization/units.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+
+import numpy as np
+
+
+def quantity_support():
+    """
+    Enable support for plotting `astropy.units.Quantity` instances in
+    matplotlib.
+
+    May be (optionally) used with a ``with`` statement.
+
+      >>> import matplotlib.pyplot as plt
```

Comment:
I think travis is failing as this needs to be skipped when matplotlib is not availble.